package.json

```
@@ -1,50 +1,50 @@
 {
   "name": "tree-sitter-redscript",
-  "version": "0.0.1",
-  "main": "bindings/node/index.js",
-  "types": "bindings/node",
-  "author": "jekky",
+  "version": "0.1.0",
+  "description": "Redscript grammar for tree-sitter",
+  "repository": "https://github.com/tree-sitter/tree-sitter-redscript",
   "license": "MIT",
-  "scripts": {
-    "build": "tree-sitter generate --no-bindings",
-    "install": "node-gyp-build",
-    "postinstall": "node-gyp configure && node-gyp build",
-    "prebuildify": "prebuildify --napi --strip"
+  "author": {
+    "name": "Jekky"
```

```suggestion
    "name": "jekky"
```
and you can list yourself